code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1492,7 +1492,12 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	hit_area = affecting.name
 	var/def_zone = affecting.body_zone
 
-	var/armor_block = H.run_armor_check(affecting, I.armortype, "<span class='notice'>Your armor has protected your [hit_area]!</span>", "<span class='warning'>Your armor has softened a hit to your [hit_area]!</span>",I.armour_penetration)
+	var/armor_block
+	switch(I.damtype)
+		if(RED_DAMAGE, WHITE_DAMAGE, BLACK_DAMAGE, PALE_DAMAGE, MELEE, BULLET, LASER, ENERGY, BOMB, BIO, RAD, FIRE, ACID)
+			armor_block = H.run_armor_check(affecting, I.damtype, "<span class='notice'>Your armor has protected your [hit_area]!</span>", "<span class='warning'>Your armor has softened a hit to your [hit_area]!</span>",I.armour_penetration)
+		else
+			armor_block = 0
```

Comment:
Is this needed..?
Also, this reminds me, but that's why armortype was kept - BULLET is armor, but there's no BULLET damage, so if it's every somehow used - it will break.

Reply:
The changes I've made recently should resolve this issue via directly converting the damages to what should be a fairly relevant type, or I can convert them to BRUTE by default otherwise.